TelepresenceBot/core/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.10.19'
+    testCompile 'org.mockito:mockito-all:1.10.19'
+}
+
+sourceCompatibility = "1.7"
+targetCompatibility = "1.7"
```

Comment:
Now you have seen it with speech marks 👅 this file was automatically generated like this. As I am such a nice guy I have changed it to appease your OCD 👅 	c654bdd